app/Http/Controllers/PostReplyController.php

```diff
@@ -105,9 +105,10 @@ public function update(PostReplyRequest $request, PostReply $postReply)
     public function destroy(PostReply $postReply)
     {
         $this->authorize('delete', PostReply::class);
+
         $post = $postReply->post_id;
         $postReply->delete();
```

Comment:
I can't see where exactly the fix is for the problem you described, @milack68. ("if a user deletes a reply the whole post and all its replies are deleted"). Did this really happen? Within this method only the postReply gets deleted as it should be, right?

Reply:
The problem was in the blade file: by clicking on the delete button for deleting a reply the modal for delete post opend.